docs/_source/guides/llms/practical_guides/fine_tune.md

```diff
@@ -818,3 +821,135 @@ output_text = tokenizer.decode(outputs[0])
 print(output_text)
 # Yes it is, toads are a sub-classification of frogs.
 ```
+
+### Chat Completion
+
+#### Background
+
+With the rise of chat-oriented models under OpenAI's ChatGPT, we have seen a lot of interest in the use of LLMs for chat-oriented tasks. The main difference between chat-oriented models and the other LLMs is that they are trained on a differently formatted dataset. Instead of using a dataset of prompts and responses, they are trained on a dataset of conversations. This allows them to generate responses that are more conversational in nature. OpenAI is one framework that [supports this kind of training](https://openai.com/blog/gpt-3-5-turbo-fine-tuning-and-api-updates).
```

Comment:
```suggestion
With the rise of chat-oriented models under OpenAI's ChatGPT, we have seen a lot of interest in the use of LLMs for chat-oriented tasks. The main difference between chat-oriented models and the other LLMs is that they are trained on a differently formatted dataset. Instead of using a dataset of prompts and responses, they are trained on a dataset of conversations. This allows them to generate responses that are more conversational in nature. And, OpenAI does support fine-tuning LLMs for chat-completion use cases. More information at https://openai.com/blog/gpt-3-5-turbo-fine-tuning-and-api-updates.
```